tests/nested/core20/kernel-reseal/task.yaml

```diff
@@ -0,0 +1,29 @@
+summary: Check that a kernel refresh reseals
+
+systems: [ubuntu-20.04-64]
+
+execute: |
+  # shellcheck source=tests/lib/nested.sh
+  . "$TESTSLIB/nested.sh"
+
```

Comment:
you may need something like https://github.com/snapcore/snapd/blob/88e460537dbcb51031ef93ccac828724fca18d98/tests/nested/core20/kernel-failover/task.yaml#L29-L39 before running `snap wait`, otherwise the test could fail on not finding the snap command